### PR TITLE
Add HMS Ewon Flexy DTDL Models

### PR DESCRIPTION
**Pull request by alexjhawk**
### Company Info
HMS Networks
https://www.hms-networks.com/

### IoT Plug and Play Device Info
HMS Networks Ewon Flexy
- with Azure Connector application installed

https://www.ewon.biz/products/ewon-flexy

### Model Submission Goals
Device certification
Presence in the Certified Device catalog

### Code Owners & Reserved Names

### CODE REVIEW

File: dtmi/com/hms_networks/ewon_flexy_azure_connector_basic-1.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	"@context": "dtmi:dtdl:context;2",
+	"@id": "dtmi:com:hms_networks:Ewon_Flexy_Azure_Connector_Basic;1",
+	"@type": "Interface",
+	"displayName": "Ewon_Flexy_Azure_Connector_Basic",
+	"contents": [
+		{
+			"@type": "Property",
+			"name": "DeviceMode",
+			"writable": true,
+			"schema": "string"
```
Comment (rido-min) [inline dtmi/com/hms_networks/ewon_flexy_azure_connector_basic-1.json:11]: suggestion. Have you considered using an `Enum` to describe the expected values for DeviceMode ?

File: dtmi/com/hms_networks/ewon_flexy_azure_connector_example-1.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+			"schema": "string"
+		},
+		{
+			"@type": "Telemetry",
+			"name": "ConnectorControl",
+			"schema": "boolean"
+		},
+		{
+			"@type": "Telemetry",
+			"name": "EXAMPLE",
```
Comment (rido-min) [inline dtmi/com/hms_networks/ewon_flexy_azure_connector_example-1.json:20]: What means `EXAMPLE` ? In this repo we'd like to keep *production ready* interfaces, and not examples.

**Review by rido-min — CHANGES_REQUESTED** [review]
Hi @alexjhawk,

Thanks for your submission, I've added a couple of suggestions to review before approving.

File: dtmi/com/hms_networks/ewon_flexy_azure_connector_example-1.json (hunk 2 above)
Comment (alexjhawk) [reply dtmi/com/hms_networks/ewon_flexy_azure_connector_example-1.json]: Hi, I have removed this example interface. This PR now includes only _production ready_ interfaces.

File: dtmi/com/hms_networks/ewon_flexy_azure_connector_basic-1.json (hunk 1 above)
Comment (alexjhawk) [reply dtmi/com/hms_networks/ewon_flexy_azure_connector_basic-1.json]: I read the documentation for `Enum`s and have updated the blank and basic models accordingly. There are only two possible values, the string "0" and string "1". Thank you for the suggestion.

**Comment by alexjhawk** [comment]
Updated to rebase with the latest changes in branch `main`

**Review by digimaun — APPROVED** [review]

**Pull request merged**